README.md

```diff
@@ -1,10 +1,55 @@
 # MuckRock
+**MuckRock** &middot; [Squarelet][squarelet] &middot; [DocumentCloud][documentcloud] &middot; [DocumentCloud-Frontend][documentcloudfrontend]
 
 [![Codeship Status for MuckRock/muckrock][codeship-img]][codeship]
 [![codecov.io][codecov-img]][codecov]
 
 MuckRock is a non-profit collaborative news site that gives you the tools to keep our government transparent and accountable.
 
+## Prerequisites 
+MuckRock depends on Squarelet for user authentication. As the services need to communivate directly, the development environment for MuckRock depends on the development environment for Squarelet - the MuckRock docker containers will join Squarelet's docker network. [Please install Squarelet and set up its development environment first][squarelet].
+
+### Squarelet Integration
```

Comment:
I think this section could be lifted to the end of the Squarelet section.

If you are only working on Squarelet, you will still need a superuser.

If you are working on both MuckRock and DocumentCloud, you only need to create a single superuser on Squarelet, so having it at the end of the Squarelet instructions should make more sense than having it duplicated.

Reply:
completely agree, was removed and moved